eng/common/scripts/update-docs-metadata.ps1

```diff
@@ -24,19 +24,19 @@ Write-Host "> $PSCommandPath $args"
 function GetMetaData($lang){
   switch ($lang) {
     "java" {
-      $metadataUri = "https://raw.githubusercontent.com/Azure/azure-sdk/master/_data/releases/latest/java-packages.csv"
+      $metadataUri = "https://raw.githubusercontent.com/Azure/azure-sdk/master/_data/allpackages/java-packages.csv"
```

Comment:
Why switch to allpackages? 

Also as a heads up the location of the csv files may be moving with PR https://github.com/Azure/azure-sdk/pull/1341/files.